public/_includes/_scripts-minimum.jade

```diff
@@ -18,6 +18,16 @@ script.
 
```

Comment:
can you remove the swifty script? we shouldn't load it if we are not using it any more.